src/skmultiflow/meta/leverage_bagging.py

```diff
@@ -281,17 +282,14 @@ def __partial_fit(self, X, y):
                     self.ensemble[i].partial_fit(X=np.asarray([X]), y=np.asarray([y_coded]),
                                                  classes=classes)
 
-            try:
-                pred = self.ensemble[i].predict(np.asarray([X]))
-                if pred is not None:
-                    add = 1 if (pred[0] == y_coded) else 0
-                    error = self.adwin_ensemble[i].estimation
-                    self.adwin_ensemble[i].add_element(add)
-                    if self.adwin_ensemble[i].detected_change():
-                        if self.adwin_ensemble[i].estimation > error:
-                            change_detected = True
-            except ValueError:
-                change_detected = False
```

Comment:
Do not we expect this portion to throw a `ValueError` exception anymore? Could you elaborate on this change too?

Reply:
Sorry about that. This is an obsolete check from the time when not all models were defaulting to `prediction=[0]` when empty. This is no longer the case as now all `scikit-multiflow` classifiers return a valid value.

Reply:
Thank you for the clarification!